README.md

```diff
@@ -49,10 +49,13 @@ The Make and CMake build systems create three libraries: libmbedcrypto, libmbedx
 
 You need the following tools to build the library with the provided makefiles:
 
-* GNU Make or a build tool that CMake supports.
-* A C99 toolchain (compiler, linker, archiver). We actively test with GCC 5.4, Clang 3.8, IAR8 and Visual Studio 2013. More recent versions should work. Slightly older versions may work.
-* Python 3 to generate the test code, and to generate sample programs in the development branch.
+* GNU Make 3.82 or a build tool that CMake supports.
+* A C99 toolchain (compiler, linker, archiver). We actively test with GCC 5.4, Clang 3.8, IAR 8 and Visual Studio 2013. More recent versions should work. Slightly older versions may work.
+* Python 3.6 to generate the test code, and to generate sample programs in the development branch.
 * Perl to run the tests, and to generate some source files in the development branch.
+* CMake 3.10.2 or later (if using CMake).
+* Microsoft Visual Studio 2013 or later (if using Visual Studio).
+* Doxygen 1.8.11 or later (if building the documentation; slightly older versions should work).
```

Comment:
Note: currently some later versions don't work as they give more warnings, which result in some of our scripts failing. However, I think we should still document as "1.8.11 or later", and if that's not the case, then that's a bug: https://github.com/ARMmbed/mbedtls/issues/3497

Reply:
Recent versions of Doxygen do work, actually — I used a misleading title in #3497. They emit warnings, which causes our CI script to fail. But those warnings don't affect the rendering, or only marginally, so from the perspective of users, running Doxygen does work.

Reply:
Yes, from a user's perspective, `make apidoc` works. From my perspective, `tests/scripts/all.sh check_doxygen_warnings` fails on my machine and this annoys me because I like to use `tests/scripts/all.sh 'check_*'` as a quick sanity check before pushing. Anyway.